src/pages/Main/Main.js

```diff
@@ -10,25 +11,29 @@ const Main = () => {
       .then(response => response.json())
       .then(result => setMainData(result));
   }, []);
+
   return (
-    <div className="mainForm">
-      {mainData.map(data => (
-        <Link to={data.url} key={data.id}>
-          <div key={data.id} id={data.id}>
-            <div className="formBlock">
-              {data.video ? (
-                <video src={data.video} loop autoPlay muted />
-              ) : (
-                <img src={data.image} />
-              )}
-              <p className="title">{data.title} </p>
-              <p className="mainText">{data.mainText}</p>
-              <p className="mainText2">{data.mainText2}</p>
-              <button> 구매하기</button>
+    <div>
+      <div className="mainForm">
+        {mainData.map(data => (
+          <Link to={data.url} key={data.id}>
+            <div key={data.id} id={data.id}>
+              <div className="formBlock">
+                {data.video ? (
+                  <video src={data.video} loop autoPlay muted />
+                ) : (
+                  <img src={data.image} />
+                )}
```

Comment:
삼항연산이 거슬리신다면 이런식으로 구현해도 좋습니다.
```suggestion
<Thumbnail video={data.video} image={data.image}>
```

```jsx
const Thumbnail = ({ video, image }) => {
  if(video) return <video src={data.video} loop autoPlay muted />
  
  return <img src={data.image} />
}
```

Reply:
이름은 Thumbnail 도되고 Card 로 해도 될 듯 합니다!

Reply:
@YunKukPar Thumbnail 컴포넌트로 분리해서 수정했습니다!!!!!!!